Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/MinimumFlowBypass/Subsequences/FlowSetpoint.mo

```diff
@@ -0,0 +1,693 @@
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid,
+          horizontalAlignment=TextAlignment.Right,
+          textString="Minimum flow setpoint when"),
+        Text(
+          extent={{-50,-64},{14,-78}},
+          lineColor={0,0,127},
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid,
+          horizontalAlignment=TextAlignment.Right,
+          textString="enabling additional chiller"),
+        Text(
+          extent={{-38,-102},{14,-116}},
+          lineColor={0,0,127},
+          fillColor={215,215,215},
+          fillPattern=FillPattern.Solid,
+          horizontalAlignment=TextAlignment.Right,
+          textString="disabling one chiller")}),
+  Documentation(info="<html>
+<p>
+Block that output chilled water minimum flow setpoint for primary-only
+plants with a minimum flow bypass valve, 
+according to ASHRAE RP-1711 Advanced Sequences of Operation for HVAC Systems Phase II –
+Central Plants and Hydronic Systems (Draft 4 on March 26, 2019), 
+section 5.2.8 Chilled water minimum flow bypass valve.
+</p>
+<p>
+1. For plants with parallel chillers, bypass valve shall modulate to maintain minimum
+flow as measured by the chilled water flow meter at a setpoint that ensures minimum
+flow through all operating chillers, as follows:
+</p>
+<ul>
+<li>
+For the operating chillers in current stage, identify the chiller with the 
+highest raio of <code>minFloSet</code> to <code>maxFloSet</code>.
+</li>
+<li>
+Calculate the minimum flow setpoint as the highest ratio multiplied by the sum
+of <code>maxFloSet</code> for the operating chillers.
+</li>
+</ul>
+<table summary=\"summary\" border=\"1\">
+<tr>
+<th> Chiller </th> 
+<th> Minimum flow </th>  
+<th> Maximum flow </th>  
+</tr>
+<tr>
+<td align=\"center\">1</td>
+<td align=\"center\"><code>minFloSet</code>[1]</td>
+<td align=\"center\"><code>maxFloSet</code>[1]</td>
+</tr>
+<tr>
+<td align=\"center\">2</td>
+<td align=\"center\"><code>minFloSet</code>[2]</td>
+<td align=\"center\"><code>maxFloSet</code>[2]</td>
+</tr>
+<tr>
+<td align=\"center\">...</td>
+<td align=\"center\">...</td>
+<td align=\"center\">...</td>
+</tr>
+</table>
+<br/>
+
+<p>
+2. For plants with series chillers, bypass valve shall modulate to maintain minimum
+flow as measured by the chilled water flow meter at a setpoint equal to the largest
+<code>minFloSet</code> of the operating series chillers.
+</p>
+<p>
+3. If there is any stage change requiring a chiller on and another chiller off,
+the minimum flow setpoint shall temporarily change to account for the 
+<code>minFloSet</code> of both enabling and disabling chillers prior to starting
+the newly enabled chiller.
+</p>
+<p>
+Note that when there is stage change thus requires changes of 
+minimum bypass flow setpoint, the change should be slowly.
+For example, this could be accomplished by resetting the setpoint X GPM/second, 
+where X = (NewSetpoint - OldSetpoint) / <code>byPasSetTim</code>.
```

Comment:
@AntoineGautier here intends to change the setpoint slowly. Say changing the setpoint from old one to new one, it should be slow. Otherwise, it would cause hydraulic instability. 
This one did not explicitly shown in this document but is shown in other document sent by Taylor Enginnering. I may double check with them.